skyportal/handlers/api/photometry.py

```diff
@@ -398,27 +397,6 @@ def get_values_table_and_condition(self, df):
            `df` against the Photometry table using the deduplication index.
         """
 
-        # https://github.com/sqlalchemy/sqlalchemy/wiki/PGValues
-        class _photometry_values(FromClause):
-            """Render a postgres VALUES statement (in-memory constant table)."""
-
-            named_with_column = True
-
-            def __init__(self, columns, *args, **kw):
-                self._column_args = columns
-                self.list = args
-                self.alias_name = self.name = kw.pop("alias_name", None)
-
-            def _populate_column_collection(self):
-                for c in self._column_args:
-                    c._make_proxy(self)  # noqa
-
-            @property
-            def _from_objects(self):
-                return [self]
-
-        # https://github.com/sqlalchemy/sqlalchemy/wiki/PGValues
-        @compiles(_photometry_values)
         def _compile_photometry_values(element, compiler, asfrom=False, **kw):
```

Comment:
Is this actually used anywhere now either?

Reply:
Good point. Removing.